Workbooks/SapMonitor2.0/Alerts/Create Alert rule (step 1 of 2)/Create Alert rule (step 1 of 2).workbook

```diff
@@ -379,13 +379,21 @@
     {
       "type": 1,
       "content": {
-        "json": "### Choose a name, provider instance and severity for the alert"
+        "json": "### Choose a Name, severity for the alert"
```

Comment:
"Choose a name and severity for the alert" would be better

Reply:
fixed